api/volume.go

```diff
@@ -388,8 +388,13 @@ func (s *Server) PVCreate(rw http.ResponseWriter, req *http.Request) error {
 		return fmt.Errorf("cannot create PV for standby volume %v", vol.Name)
 	}
 
+	// if PV fstype is not defined, default to ext4
+	if input.PVFsType == "" {
```

Comment:
Let the manager decide the default FSType.

Reply:
moved `fsType` check to manager part.